.coveragerc

```diff
@@ -6,6 +6,8 @@ omit =
     env/*
     */test*
     */environment.py
+    run.py
```

Comment:
What does this line do?

Reply:
Because run.py only creates a grpc connection to the server and creates an entry point to the project. you don't need a test file because it is tested in the integration test